mediorum/server/serve_blob.go

```diff
@@ -254,7 +254,7 @@ func (s *MediorumServer) requireSignature(next echo.HandlerFunc) echo.HandlerFun
 			}
 
 			// check signature not too old
-			age := time.Since(time.Unix(sig.Data.Timestamp, 0))
+			age := time.Since(time.Unix(sig.Data.Timestamp/1000, 0))
```

Comment:
This PR checks out, test case showing `Fri Jul 31 2020 01:32:03`

However, I do think this logic is kind of error prone since it depends on the correct sysclock here and the test case is somewhat brittle because of that too.

Reply:
not blocking!

Reply:
I do think the test case should be less brittle - more like "take in these variables, sign it, and then do the request", but that's a little more involved.

what's wrong with this age logic tho? wouldn't any timestamp verification have to rely on the system clock being accurate?

Reply:
yeh